torch/_decomp/decompositions.py

```diff
@@ -1218,3 +1218,60 @@ def stack(tensors: List[Tensor], dim: int = 0) -> Tensor:
         return out.view(result_sizes)
     else:
         return torch.cat(get_stack_inputs(tensors, wrapped_dim), dim)
+
+
+@register_decomposition(aten.frac)
+def frac(input: Tensor) -> Tensor:
+    return input - torch.trunc(input)
+
+
+@register_decomposition(aten.celu)
+def celu(input: Tensor, alpha: float = 1.0) -> Tensor:
+    inv_alpha = 1.0 / alpha
```

Comment:
@Chillee, @ngimel 

Does it make a difference if I write "1 / alpha" here?

In GraphModule produced by make_fx(), 1.0 is converted into Tensor(1.0, dtype=torch.float64) 
Instead, 1 is converted into Tensor(1, dtype=torch.int64) 

Such difference could have different result after going thought type promotion?

Reply:
for example, 

>>> torch.promote_types(torch.float32, torch.float64)
torch.float64
>>> torch.promote_types(torch.float32, torch.int64)
torch.float32

Reply:
I think, no, since in this case, it's purely an operation between two floats - so 1.0/alpha will always result in a float.

Reply:
it's slightly different no ?

```
>>> (torch.tensor(1.0, dtype=torch.float) + torch.tensor(1.0, dtype=torch.float64)).dtype
torch.float64
>>> (torch.tensor(1.0, dtype=torch.float) + 1.0).dtype
torch.float32
```

Reply:
Both alpha and one are floats (not tensors), so it doesn’t matter.